app/src/editor/canvas/components/ModuleRenderer/modules.js

```diff
@@ -0,0 +1,173 @@
+// @flow
+
```

Comment:
Hrm, this list is yet another thing we need to manually update if a module is added or removed. FYI we're already a few steps out of sync with what's available on the backend. Can this stay dynamic like it is in `ModuleHelp`? If not, this should probably live near where these are defined in the docs.

update: ahh it's only used in the story. Hrm, maybe it should live with the story then, otherwise someone might be tempted to import this instead of using the webpack dynamic import thing.